src/components/Tooltip/tooltip.module.scss

```diff
@@ -74,4 +73,16 @@ $tooltip-arrow-shadow: 1px 1px 2px rgba(15, 20, 31, 0.12);
         animation: scaleTooltip $motion-duration-extra-fast
             $motion-easing-easeinout 0s;
     }
+
+    &.small {
+        max-width: 140px;
+    }
+
+    &.medium {
+        max-width: 240px;
+    }
+
+    &.large {
+        max-width: 360px;
+    }
```

Comment:
is there a default, or it will be allowed to grow as large as necessary?

Reply:
good catch default should be small, making that change